Documentation/contributing/development/debugging.rst

```diff
@@ -40,7 +40,7 @@ implementations, notably musl which is common in Alpine Linux images, terminate
 the whole DNS search in these cases. This often manifests as a connect error in
 applications, as the libc lookup returns no data.
 To work around this, denied responses can be configured to be NXDOMAIN by
-setting the ``--tofqdns-dns-reject-response-code`` command line argument.
+setting ``--tofqdns-dns-reject-response-code=nameError`` on the command line.
```

Comment:
I don't understand the change here or how it relates to the PR description. `--tofqdns-dns-reject-response-code=nameError` is a legitimate configuration so changing this line provides strictly less information to users trying to work around this issue.

Reply:
I thought the documentation related changes were harmless enough to include in this PR but it seem that my assumption was wrong, sorry for that. I'll make amend and create a separate PR for them.

Reply:
https://github.com/cilium/cilium/pull/14439